tests/sentry/grouping/ingest/test_seer.py

```diff
@@ -157,12 +165,40 @@ def test_obeys_customized_fingerprint_check(self) -> None:
             (custom_fingerprint_event, False),
             (built_in_fingerprint_event, False),
         ]:
-
             assert (
                 should_call_seer_for_grouping(event, event.get_grouping_variants())
                 is expected_result
             ), f'Case with fingerprint {event.data["fingerprint"]} failed.'
 
+    @patch("sentry.grouping.ingest.seer.metrics")
+    def test_obeys_empty_stacktrace_string_check(self, mock_metrics: Mock) -> None:
+        self.project.update_option("sentry:similarity_backfill_completed", int(time()))
+        new_event = Event(
+            project_id=self.project.id,
+            event_id="12312012112120120908201304152013",
+            data={
+                "title": "title",
+                "platform": "python",
+                "stacktrace": {"in-app": False, "contributes": False, "frames": [{}]},
```

Comment:
I don't think `in-app` and `contributes` are things which show up in actual event data (only in variants). Also, an empty frames list also wouldn't get past `event_content_is_seer_eligible`... But here's maybe a simpler solution: Can we just mock `get_stacktrace_string` to return an empty string?

Reply:
The empty frame list actually passes `event_content_is_seer_eligible`, hence how the empty stacktrace metric is called after